src/main/java/Mel.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Scanner;
+
+public class Mel {
+
+    public static final String DRAW_HORIZONTAL_LINE = "\t________________________________________";
+    public static final int MARK_WORD_LEN = 4;
+    public static final int UNMARK_WORD_LEN = 6;
+    public static final int INPUT_SPACE_BUFFER = 2;
```

Comment:
what is the purpose of INPUT_SPACE_BUFFER?

Reply:
The purpose of the buffer is the space and first digit that follows the commands